src/components/PageIntro.jsx

```diff
@@ -1,5 +1,6 @@
 export default function PageIntro(prop) {
```

Comment:
- It will be better to name `prop` => `props`

Reply:
I didn't build this component , i just i used it .
But it's ok , i will fix it .